WikiWatershed/model-my-watershed
# Remove port 8080 from firewall rules

## Overview

We were previously accessing the Cinergi API on port 8080. After their recent production deployment, we can now access the API over port 80. This PR disallows ingress/egress traffic on port 8080 from the AppServer and NAT Security Groups since it's no longer necessary.

Connects #2144 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

 * This change has already been deployed to Staging. Check the security group rules for [sgNAT](https://console.aws.amazon.com/vpc/home?region=us-east-1#securityGroups:filter=nat) and [sgAppServer](https://console.aws.amazon.com/vpc/home?region=us-east-1#securityGroups:filter=sgAppServer$) ensure that there are no references to port 8080.
